app/controllers/admin/users_controller.rb

```diff
@@ -23,49 +23,43 @@ def edit
     end
 
     def update
-      render_edit and return unless entry.valid?
-
-      handle_service_result(user_update,
-                            -> { redirect_to admin_users_path },
-                            -> { render_edit })
+      if user.update(user_params)
```

Comment:
Did you write tests for this?

Please use `update_attributes`. See this http://apidock.com/rails/ActiveRecord/Base/update/class and lets discuss if you don't understand why you don't want to use `update` :smile: